File: swing/src/main/java/wannabe/swing/renderer/FilledThreeDSquare.java
package wannabe.swing.renderer;

import java.awt.Graphics;
import wannabe.swing.SwingProjected;

public class FilledThreeDSquare extends SwingRenderer {
  @Override public void draw(Graphics g, SwingProjected p) {
    g.fill3DRect(p.left, p.top, p.size, p.size, true);
  }
}

File: swing/src/main/java/wannabe/swing/renderer/FilledThreeDSquare.kt
package wannabe.swing.renderer

import wannabe.swing.SwingProjected
import java.awt.Graphics

class FilledThreeDSquare : SwingRenderer() {
  override fun draw(
    g: Graphics,
    p: SwingProjected
  ) {
    g.fill3DRect(p.left, p.top, p.size, p.size, true)
  }
}
